core/src/main/python/wlsdeploy/tool/create/domain_creator.py

```diff
@@ -345,6 +345,12 @@ def __create_domain(self):
         self.library_helper.install_domain_libraries()
         self.library_helper.extract_classpath_libraries()
         self.wlsroles_helper.process_roles()
+        if os.environ.has_key('__STORE_MODEL__'):
```

Comment:
Any interest in changing this to "WLSDEPLOY_STORE_MODEL"?  They do this elsewhere in the code, such as WLSDeployLoggingConfig.java .  I could live with either, though